lib/rules/validate-quote-marks.js

```diff
@@ -106,10 +106,16 @@ module.exports.prototype = {
                     return;
                 }
 
+                token.value = token.value.replace(
+                    new RegExp('^' + mark + '(.*)' + mark + '$'),
```

Comment:
Can't get how ruin this ;-) You know string without `'` or `"`? ES6 template strings?